frontend/src/components/modals/player_update_modal.tsx

```diff
@@ -46,7 +47,7 @@ export default function PlayerUpdateModal({
       <Modal opened={opened} onClose={() => setOpened(false)} title={t('edit_player')}>
         <form
           onSubmit={form.onSubmit(async (values) => {
-            await updatePlayer(tournament_id, player.id, values.name, values.active, null);
+            await updatePlayer(tournament_id, player.id, values.name, values.active, null, values.elo_score);
```

Comment:
I'd refactor this function to accept a single object argument if this were my codebase, but I am trying to be consistent so I didn't do that here.